interop-future/shared/src/main/scala/scalaz/zio/interop/future.scala

```diff
@@ -54,6 +54,24 @@ object future {
       }
   }
 
+  case class CompositeException(errors: List[Throwable])
```

Comment:
I think we should consistently use `FiberFailure` tbh

Reply:
Right! Forgot about that. Thanks.